project_resource_calendar/views/equipment_calendar_view.xml

```diff
@@ -50,16 +50,31 @@
         </field>
     </record>
     
-    <record id="view_calendar_event_calendar" model="ir.ui.view">
+   <record id="view_calendar_event_calendar" model="ir.ui.view">
```

Comment:
@eilst : [Suggestion] The Id string avoid the  redundant word e.g. view_event_calendar (Refer format from other files)

Reply:
Actually, the it's the id of one existing record, from calendar module